src/rgw/CMakeLists.txt

```diff
@@ -372,6 +310,214 @@ set_target_properties(radosgw PROPERTIES OUTPUT_NAME radosgw VERSION 2.0.0
   SOVERSION 2)
 install(TARGETS radosgw DESTINATION ${CMAKE_INSTALL_LIBDIR})
 
+set(rados_sal_srcs
+  services/svc_finisher.cc
+  services/svc_bi_rados.cc
+  services/svc_bilog_rados.cc
+  services/svc_bucket.cc
+  services/svc_bucket_sobj.cc
+  services/svc_bucket_sync_sobj.cc
+  services/svc_cls.cc
+  services/svc_config_key_rados.cc
+  services/svc_mdlog.cc
+  services/svc_meta.cc
+  services/svc_meta_be.cc
+  services/svc_meta_be_otp.cc
+  services/svc_meta_be_sobj.cc
+  services/svc_notify.cc
+  services/svc_otp.cc
+  services/svc_quota.cc
+  services/svc_sync_modules.cc
+  services/svc_rados.cc
+  services/svc_sys_obj.cc
+  services/svc_sys_obj_cache.cc
+  services/svc_sys_obj_core.cc
+  services/svc_tier_rados.cc
+  services/svc_user.cc
+  services/svc_user_rados.cc
+  services/svc_zone.cc
+  services/svc_zone_utils.cc
+  rgw_bucket.cc
+  rgw_cr_rados.cc
+  rgw_d3n_datacache.cc
+  rgw_data_sync.cc
+  rgw_gc.cc
+  rgw_gc_log.cc
+  rgw_lua_request.cc
+  rgw_notify.cc
+  rgw_obj_manifest.cc
+  rgw_object_expirer_core.cc
+  rgw_period_history.cc
+  rgw_period_puller.cc
+  rgw_pubsub.cc
+  rgw_putobj_processor.cc
+  rgw_rados.cc
+  rgw_reshard.cc
+  rgw_rest_bucket.cc
+  rgw_rest_log.cc
+  rgw_rest_pubsub.cc
+  rgw_rest_pubsub_common.cc
+  rgw_rest_realm.cc
+  rgw_rest_s3.cc
+  rgw_sal_rados.cc
+  rgw_service.cc
+  rgw_sync.cc
```

Comment:
until future refactoring, I guess keeping all of sync below the line makes sense